tests/TestHelpers/SetupHelper.php

```diff
@@ -936,14 +528,7 @@ public static function getSystemConfig(
 
 		$args[] = '--no-ansi';
 
-		return self::runOcc(
-			$args,
```

Comment:
`$args[] = 'config:system:get';` this can also be removed I think

Reply:
`getSystemConfig` method is completely removable. :smile: